meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -21,7 +22,7 @@ mkdir -p /data_disk/docker
 mkdir -p /data_disk/resin-data
 
 # Start docker with the created image.
-docker daemon -g /data_disk/docker -s btrfs &
+docker daemon -g /data_disk/docker &
```

Comment:
But I was expecting that the resulting image (data_disk.img) to be aufs specific. Is is the same now whatever is the default storage driver?